to.etc.domui/src/main/java/to/etc/domui/server/AppFilter.java

```diff
@@ -189,7 +197,7 @@ public synchronized void init(final FilterConfig config) throws ServletException
 
 		initLogConfig(approot, config.getInitParameter("logpath"));
 
-		if(DeveloperOptions.isDeveloperWorkstation() ) {
+		if(DeveloperOptions.isDeveloperWorkstation()) {
```

Comment:
Can you check your IntelliJ settings? There is still a lot of formatting committed here, and that makes it difficult to see the real changes. You should set the save actions for automatic formatting to "off" until I have reformatted all of DomUI.